code/week2/operations.h

```diff
@@ -0,0 +1,9 @@
+#ifndef OPERATIONS_H
```

Comment:
are we not going to tell them about `#pragma once`? This is pretty much the biggest reason to use cpp over c.